csharp/GSDK_CSharp_Standard/InternalSdk.cs

```diff
@@ -98,6 +98,20 @@ public void Start(bool debugLogs = false)
             _signalHeartbeatEvent.Reset();
             TransitionToActiveEvent.Reset();
 
+            Task.Run(() => StartAsync($"http://{heartbeatEndpoint}/v1/metrics/{serverId}/gsdkinfo"));
```

Comment:
Same comment as above in the C++ GSDK--could we just say `_heartbeatTask = Task.Run(StartAsync(...));` and change `StartAsync` to start the heartbeat loop at the end? This way we aren't creating a Task within a Task (which is less impactful in C#, admittedly)